docs/ibis-for-dplyr-users.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>Small typo in <code>introudction</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
<p>Good catch. Fixed.</p>